packages/client/katello-host-tools/katello-host-tools.spec

```diff
@@ -179,7 +181,7 @@ rm %{buildroot}%{katello_libdir}/agent/goferd/legacy_plugin.py
 %endif
 %endif
 
-%if %{dnf_install} || %{yum_install}
+%if %{dnf_install} || %{yum_install} || %{zypper_install}
```

Comment:
Is there a build that does not trigger one of these at this point?

Reply:
There isn't now, but dnf_install will be removed as part of the 3.4 release, so it seemed simpler to just add 1 condition and remove one later than remove the entire check